mmocr/utils/__init__.py

```diff
@@ -9,6 +9,7 @@
 from .img_util import drop_orientation, is_not_png
 from .lmdb_util import lmdb_converter
 from .logger import get_root_logger
+from .ocr import MMOCR
 from .string_util import StringStrip
```

Comment:
Try switching their order to pass the test (could avoid circular import)

Reply:
I tried to change it but when I try to commit the change `isort` automatically changes it
```
flake8...................................................................Passed
seed isort known_third_party.............................................Passed
isort....................................................................Failed
- hook id: isort
- files were modified by this hook

Fixing mmocr\utils\__init__.py
```